### PR TITLE
added worst fit memory allocation technique for cpp, c, python & java closes #1492

### LINKED ISSUE #1492: Worst Fit Memory Allocation Algorithm in C, C++, Java & Python
**Is your feature request related to a problem? Please describe.**
Memory allocation techniques are an important part of the operating systems. Worst fit is one of them. In worst fit memory allocation, each new request is allocated the largest available block of memory

**Describe the solution you'd like**
I would add C, C++, Java & Python Implementation of the algorithm

### PR DESCRIPTION
**Pull request by ranjanmangla1**
## 🛠️ Issue #1492 

Issue no #1492 

## 👨‍💻 Changes proposed

Added worst fit algorithm in c, cpp, java & python

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

#### Cpp
image
#### C
image
#### Java
image
#### Python
image

### CODE REVIEW

**Review by 072arushi — CHANGES_REQUESTED** [review]
complete description

**Comment by ranjanmangla1** [comment]
> complete description

Can you tell me for what you need a complete description?

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
remove fcfs from this PR

File: C++/Job Scheduling/fcfs.cpp
Comment (zugzwang03) [inline C++/Job Scheduling/fcfs.cpp:1]: remove this file

Comment (ranjanmangla1) [reply C++/Job Scheduling/fcfs.cpp:1]: @zugzwang03 deleted it

**Review by Arun9739 — CHANGES_REQUESTED** [review]
@ranjanmangla1 Add output screenshots

**Review by zugzwang03 — APPROVED** [review]

**Comment by ranjanmangla1** [comment]
> @ranjanmangla1 Add output screenshots

Done. Added Output Screenshots @Arun9739. Please review this pull request @Kumar-laxmi @072arushi @Arun9739

**Review by Arun9739 — APPROVED** [review]

**Review by 072arushi — APPROVED** [review]

**Review by Kumar-laxmi — APPROVED** [review]

**Pull request merged**